content/docs/command-reference/remote/modify.md

```diff
@@ -481,6 +481,12 @@ $ export AZURE_PASSWORD='mysecret'
 > [env vars](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.environmentcredential)
 > available.
 
+Also some options can be propagated from the azure config file (which is managed
```

Comment:
Question: `some options` - what does it mean?

-------

I think we should describe all possible source and their priorities at the very beginning of the section. No chances people will get to this last paragraph. And if I understand this correctly (this files serves the same role as AWS credentials), this seems to be an important information that we want and should mention from the very beginning. 

Something like: there are a few ways to define config values, in the order of their priority:

1. config files
2. env
3. DVC config


etc 


(This is just an example, I don't know the details here)

Reply:
> Question: some options - what does it mean?

https://github.com/iterative/dvc.org/pull/2703#discussion_r689458522

Reply:
Thanks! That comment if you put it somewhere at the top of the document would be better - it's explicit, it explains the order clearly.

Reply:
> we should describe all possible source and their priorities at the very beginning

Agree. Done in 8b035c12. But it's gotten really complicated. This needs it's own guide (see https://github.com/iterative/dvc.org/issues/1859, it keeps popping up so hopefully we can prioritize it).